roles/etcd/handlers/backup.yml

```diff
@@ -57,3 +58,9 @@
   register: etcd_backup_v3_command
   until: etcd_backup_v3_command.rc == 0
   delay: "{{ retry_stagger | random + 3 }}"
+
+- name: Remove old etcd backups
+  shell:
+    chdir: "{{ etcd_backup_prefix }}"
+    cmd: "find . -name 'etcd-*' | head -n -{{ etcd_backup_retention_count }} | xargs rm -rf"
```

Comment:
Do we need to specify `-` after `head -n`?
The default value of etcd_backup_retention_count is -1 and the above command can be
`head -n --1`.
Is that expected?

Reply:
Hi @oomichi in the line below you dee the ansible when statement, causing this task to only trigger for values of 0 or greater. A double minus will never occur.

By using a minus character with -n, we tell head to take all but the last n. This is a convenient way of keeping all but the latest few backups.